core/types/gen_erigon_log_json.go

```diff
@@ -27,7 +27,7 @@ func (l ErigonLog) MarshalJSON() ([]byte, error) {
 	}
 
 	type ErigonLog struct {
-		Log Log `json:"log" gencoded:"required"`
+		Log       Log            `json:"log" gencoded:"required"`
```

Comment:
This file should not have been changed.

Reply:
Lint was messed up in it for some reason so when I called gofmt it got fixed up.